src/jit/jitstd/list.h

```diff
@@ -366,6 +368,18 @@ typename list<T, Allocator>::const_reference list<T, Allocator>::back() const
     return m_pTail->m_value;
 }
 
+template <typename T, typename Allocator>
+typename list<T, Allocator>::iterator list<T, Allocator>::backPosition()
```

Comment:
Eh, the problem with implementing our own STL. The normal way to do this is to use `rbegin` and `base` but it looks like our `jitstd` iterators lack `base`. Oh well, this will do.